applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -0,0 +1,47 @@
+[id="binding-workloads-using-sbo"]
+= Binding workloads using Service Binding Operator
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: binding-workloads-using-sbo
+
+toc::[]
+
+Application developers must bind a workload to one or more backing services using a binding secret generated for the purpose of storing information to be consumed by the workload.
+
+As an example, consider that the service you want to connect to is already exposing the binding data. You also need a workload to be used along with the `ServiceBinding` custom resource (CR). Through this `ServiceBinding` CR the workload sends a binding request with the details of the services to bind with.
+
+.Example of `ServiceBinding` CR
+----
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: my-postgresql
+spec:
+    services: <1>
+    - group: postgres-operator.crunchydata.com
+      version: v1beta1
+      kind: PostgresCluster
+      name: hippo
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-pguser-hippo
+    application: <2>
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+----
+<1> Specifies a list of service resources.
+<2> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
```

Comment:
```suggestion
<2> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec resource.
```

Reply:
It is just the embedded Podspec. I have removed the resource as that was a typo..